pdg/src/snapshots/c2rust_pdg__tests__analysis_test_pdg_snapshot.snap

```diff
@@ -529,6 +539,43 @@ g {
 }
 nodes_that_need_write = []
 
+g {
+	n[0]: &_1  _    => _2 @ bb0[5]:  fn test_shared_ref;
+	n[1]: copy n[0] => _3 @ bb0[8]:  fn test_shared_ref;
+	n[2]: copy n[1] => _5 @ bb0[12]: fn test_shared_ref;
+	n[3]: copy n[2] => _1 @ bb0[0]:  fn shared_ref_foo; 
+	n[4]: copy n[3] => _0 @ bb0[1]:  fn shared_ref_foo; 
+	n[5]: copy n[4] => _4 @ bb1[0]:  fn test_shared_ref;
+}
+nodes_that_need_write = []
+
+g {
+	n[0]: &_1 _ => _4 @ bb0[8]: fn test_unique_ref;
+}
+nodes_that_need_write = []
+
+g {
+	n[0]: &_4 _ => _3 @ bb0[10]: fn test_unique_ref;
+}
+nodes_that_need_write = []
+
+g {
+	n[0]: &_3        _    => _5 @ bb0[13]: fn test_unique_ref;
+	n[1]: addr.store n[0] => _  @ bb0[18]: fn test_unique_ref;
+}
+nodes_that_need_write = [1, 0]
+
+g {
+	n[0]: &_1 _ => _7 @ bb0[16]: fn test_unique_ref;
```

Comment:
these will be tied together properly once https://github.com/immunant/c2rust/pull/533 lands